nixos/modules/profiles/minimal.nix

```diff
@@ -13,7 +13,17 @@ with lib;
 
   documentation.enable = mkDefault false;
 
+  documentation.doc.enable = mkDefault false;
+
+  documentation.info.enable = mkDefault false;
+
+  documentation.man.enable = mkDefault false;
```

Comment:
man pages are not that big to be honest and not having any help can be quite annoying.

Reply:
In other profiles, documentation is enabled by default. In the nixos tests, this profile is used unchanged. Documentation is not needed there.

Reply:
Added man to netboot minimal image.